proto/backend/service.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright 2024 The Bucketeer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package bucketeer.backend;
+option go_package = "github.com/bucketeer-io/bucketeer/proto/backend";
+
+import "google/api/annotations.proto";
+import "google/protobuf/field_mask.proto";
+
+import "proto/feature/feature.proto";
+
+message GetFeatureRequest {
+  string environment_namespace = 1;
+  string id = 2;
+}
+
+message GetFeatureResponse {
+  feature.Feature feature = 1;
+}
+
+message UpdateFeatureRequest {
+  string comment = 1;
+  string environment_namespace = 2;
+  google.protobuf.FieldMask field_mask = 3;
```

Comment:
Where are you going to use this?

Reply:
field mask is for partial update
https://protobuf.dev/programming-guides/api/#use-update-field-mask

Reply:
@cre8ivejp I changed my mind. I removed the field mask.
Usthe ing field mbringsring complextoy of UI logic. I want to start by being straightforward.
WDYT?

Reply:
Agree. Let's try to use it later if needed.